deeppavlov/models/kbqa/tree_to_sparql.py

```diff
@@ -116,50 +120,263 @@ def make_sparse_matrix(self, words: List[str]):
 class SlovnetSyntaxParser(Component, Serializable):
     """Class for syntax parsing using Slovnet library"""
 
-    def __init__(self, load_path: str, navec_filename: str, syntax_parser_filename: str, **kwargs):
+    def __init__(self, load_path: str, navec_filename: str, syntax_parser_filename: str, tree_patterns_filename: str,
+                 lang: str = "ru", **kwargs):
```

Comment:
English version does not need syntax parser, because it uses classification model to define sparql query template, in Russian language there are not enough data for training the classification model.